src/@types/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module '*.png' {
+  const content: string
+  export default content
+}
+
+declare module '*.jpg' {
+  const content: string
+  export default content
+}
+
+declare module '*.jpeg' {
+  const content: string
+  export default content
+}
+
+declare module '*.gif' {
+  const content: string
+  export default content
+}
+
+declare module '*.svg' {
+  import React = require('react');
```

Comment:
We could use import from here like this? https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/lib/react-app.d.ts